6_drivers_prep/code/nldas_feather_utils.R

```diff
@@ -0,0 +1,84 @@
+create_feather_task_makefile <- function(makefile, feather_files, include, feather_dir, ind_dir, packages, sources){
+  cube_task_step <- create_task_step(
+    step_name = 'nccopy',
+    target = function(task_name, step_name, ...) {
+      file.path(nc_dir, task_name)
+    },
+    command = "nccopy_nldas(target_name)"
+  )
+  
+  cube_task_plan <- create_task_plan(sub_files, list(cube_task_step), final_steps='nccopy', ind_dir=ind_dir)
+  
+  create_task_makefile(
```

Comment:
for looping, a reminder that you may want to separate the task makefile target from the task plan target so you can pass them both to `loop_tasks`